spec/services/doc_auth_router_spec.rb

```diff
@@ -50,6 +50,7 @@
       end
 
       it 'doc_auth_vendor randomizes near configured level' do
+        srand(999)
```

Comment:
what if we rewrite the tests to pass in specific discriminators rather than have any randomness at all? maybe we construct them via `123.to_s(16)` to get the hex values we want?

Reply:
ex check a few key values (over, eq, under 57) than try to test all of them and count

Reply:
I'm not following super well, can you write a short example?

Reply:
I think he's saying pass in the same series of discriminators everytime, e.g.:

```ruby
iterations.times do |i|
  results.push(DocAuthRouter.doc_auth_vendor(discriminator: i.to_s(16)))
end 
```

Then, you'll get a deterministic result.